src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/GVMDependenciesNode.cs

```diff
@@ -10,6 +10,8 @@
 namespace ILCompiler.DependencyAnalysis
 {
     /// <summary>
+    /// Represents a use of a generic virtual method slot. This node only tracks
```

Comment:
It's not quite clear to me what a "slot" means in this context.

Reply:
Slot in the sense of ECMA-335 - either a newslot method, or virtual that doesn't have another virtual method with the same name/sig in the inheritance hierarchy.